### PR TITLE
[Doc] Docs review

### PR DESCRIPTION
**Pull request by Tau-J**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/migration.md [hunk 1]
```diff
@@ -434,6 +434,15 @@ init_cfg=dict(
     checkpoint='PATH/TO/YOUR_MODEL_WEIGHTS.pth'),
 ```
 
+If you want to load a checkopoint to your backbone, you should specify the `prefix`:
+
+```Python
+init_cfg=dict(
+    type='Pretrained',
+    prefix='backbone.',
+    checkpoint='PATH/TO/YOUR_CHECKPOINT.pth'),
```
[the remark this answers is not in the document]
Comment (Tau-J) [reply docs/en/migration.md:443]: Yes, you have to specify the prefix when loading a checkpoint model trained based on mmpose

[the remark this answers is not in the document]
Comment (Ben-Louis) [reply docs/en/migration.md:443]: Sorry for misleading you. I wonder if it is necessary to add the postfix `.pth`. In other docs, such as https://github.com/apchenstu/TensoRF/blob/main/README.md#rendering, the postfix of a non-specific file is often omitted.

[the remark this answers is not in the document]
Comment (ly015) [reply docs/en/migration.md:443]: I think it's OK to have `.pth` here, making it clear that this is a path to a .pth file.

**Comment by ly015** [comment]
LGTM.

**Review by ly015 — APPROVED** [review]

**Pull request merged**